test/libgit2-online.jl

```diff
@@ -7,13 +7,12 @@
 #########
 # init & clone
 mktempdir() do dir
-    repo_url = "github.com/JuliaLang/Example.jl"
-    https_prefix = "https://"
-    ssh_prefix = "git@"
+    repo_url_https = "https://github.com/JuliaLang/Example.jl"
+    repo_url_git = "git@github.com:JuliaLang/Example.jl"
```

Comment:
Original code specified the SSH URL as "git@github.com/JuliaLang/Example.jl" instead of "git@github.com:JuliaLang/Example.jl"